examples/tensorflow/text-classification/run_text_classification.py

```diff
@@ -119,6 +60,56 @@ def on_epoch_end(self, epoch, logs=None):
         self.model.save_pretrained(self.output_dir)
 
 
+def convert_dataset_for_tensorflow(dataset, non_label_column_names, batch_size, labels, dataset_mode, drop_remainder):
```

Comment:
dataset_mode and drop_remainder could have defaults here.

Reply:
Done!